equinix/resource_network_device_acc_test.go

```diff
@@ -1522,27 +1536,56 @@ resource "equinix_network_device" "%{device-resourceName}" {
 }
 
 func testAccNetworkDeviceACL(ctx map[string]interface{}) string {
-	config := nprintf(`
+	var config string
+	if _, ok := ctx["acl-name"]; ok {
+		config += nprintf(`
 resource "equinix_network_acl_template" "%{acl-resourceName}" {
   name          = "%{acl-name}"
   description   = "%{acl-description}"
   inbound_rule {
-    subnet  = "10.0.0.0/24"
+    subnet   = "10.0.0.0/24"
+    protocol = "IP"
+    src_port = "any"
+    dst_port = "any"
+  }
+}`, ctx)
+	}
+	if _, ok := ctx["mgmtAcl-name"]; ok {
+		config += nprintf(`
+resource "equinix_network_acl_template" "%{mgmtAcl-resourceName}" {
+  name          = "%{mgmtAcl-name}"
+  description   = "%{mgmtAcl-description}"
+  inbound_rule {
+    subnet   = "11.0.0.0/24"
     protocol = "IP"
     src_port = "any"
     dst_port = "any"
   }
 }`, ctx)
+	}
 	if _, ok := ctx["acl-secondary_name"]; ok {
 		config += nprintf(`
 resource "equinix_network_acl_template" "%{acl-secondary_resourceName}" {
   name          = "%{acl-secondary_name}"
   description   = "%{acl-secondary_description}"
   inbound_rule {
-     subnet  = "192.0.0.0/24"
-     protocol = "IP"
-     src_port = "any"
-     dst_port = "any"
+    subnet   = "192.0.0.0/24"
+    protocol = "IP"
+    src_port = "any"
+    dst_port = "any"
+  }
+}`, ctx)
+	}
+	if _, ok := ctx["mgmtAcl-secondary_name"]; ok {
```

Comment:
We are not using this configuration template in the tests today. Should we keep it here or remove it until we add a test that exercises it? Can we test it in this PR?

Reply:
Let's keep it just in case we forget it in the future. The logic and function for this secondary device acl is exactly same as single device. They call the same function to exercise the change.